setup.cfg

```diff
@@ -55,6 +55,7 @@ test =
     sgp4>=2.3
 all =
     scipy>=0.18
+    certifi
```

Comment:
This dependency will also need to be listed https://docs.astropy.org/en/latest/install.html .